README.md

```diff
@@ -26,3 +26,21 @@ Use the projects git hooks
 ```
 git config core.hooksPath .githooks/
 ```
+
```

Comment:
please remove this from this PR. This should have its own pr. 
I think i left a comment about this but i cant find it ;/